### DISCUSSION TITLE
As far as I can tell desktop installation just installs oracle server, Network Miner, Wireshark

### DISCUSSION THREAD

**Discussion by kronkinatorix** [Security-Onion-Solutions/securityonion#11672] [General] [answered]
There must be something I'm missing here, but the documentation on the site seems to gloss over what happens after the install. There doesn't seem to be anything installed on my system that I can use aside from the above, and I've found some suggestions to access the securty onion console through a web browser with no indication on how that might be done. 

There's an nsm directory on my root drive, no idea what that's for and the documentation doesn't seem to mention it. 

There's no so-status command.

At a complete loss as to what's next. Unless this all an elaborate joke in which case ha ha, you got me.

**Comment by petiepooo** [comment]
Did you follow step 7 of the documentation at https://docs.securityonion.net/en/2.4/installation.html?

**Reply by kronkinatorix** [reply]
I have now. 

So-status displays: 

 Installation does not appear to be complete. A highstate has not fully  completed.

I don't smoke weed anymore so I'm not sure how I'll be able to get, and I'm in canada, not the states. 

Jokes aside, the configuration for the standalone installation just directs me towards the console. Again, no mention of how to access the console.

**Comment by TOoSmOotH** [comment, marked as answer]
Security Onion Desktop is used as a complimentary piece to Security Onion. Some users install SOD and then install an import install on top of that or vice versa.
